src/Framework/README.md

```diff
@@ -0,0 +1,25 @@
+# Framework
+
+Contains packages that provide reference assembles, default sets APIs for building ASP.NET Core applications, and assets used for self-contained deployments.
+
+## Description
+
+The following contains a description of each sub-directory in the `Framework` directory.
+
+- `App.Ref.Internal`: Empty project for identifying the non-stable version of AspNetCore at the time the targeting pack was last built
+- `App.Ref`: Contains reference assemblies, documentation, and other design-time assets
+- `App.Runtime`: Provides a default set of APIs for building ASP.NET Core applications, and assets used for self-contained deployments.
+
+## Development Setup
+
+### Build
+
+To build this specific project from source, follow the instructions [on building a subset of the code](https://github.com/dotnet/aspnetcore/blob/master/docs/BuildFromSource.md#building-a-subset-of-the-code).
+
+### Test
+
+To run the tests for this project, [run the tests on the command line](https://github.com/dotnet/aspnetcore/blob/master/docs/BuildFromSource.md#running-tests-on-command-line) in this directory.
+
+## More Information
+
+For more information, see the [ASP.NET Core README](../../README.md).
```

Comment:
Why use a relative link here and HTTPS for the two links above❔ Not sure which is better but I'd rather we were consistent.

Reply:
I think this was an issue with the template that's been fixed after i copied, I'll resync